apps/nextjs/src/components/sidebar/team-switcher.tsx

```
@@ -0,0 +1,97 @@
+    teams.length > 0 ? teams[0] : null,
+  );
+
+  return (
+    <SidebarMenu>
+      <SidebarMenuItem>
+        <DropdownMenu>
+          <DropdownMenuTrigger asChild>
+            <SidebarMenuButton
+              size="lg"
+              aria-label={`Switch team (current: ${activeTeam?.name ?? "None"})`}
+              role="combobox"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)